src/components/__tests__/LibrariesListItem-test.tsx

```diff
@@ -3,7 +3,6 @@ import { stub } from "sinon";
 
 import * as React from "react";
 import * as Enzyme from "enzyme";
-import { mount } from "enzyme";
```

Comment:
What's the benefit of importing all instead of just `mount`?

Reply:
For specifying the type for the wrapper--`Enzyme.CommonWrapper`.  This was during the course of getting rid of some TypeScript error messages a while ago.  (On the other hand, there's no benefit to specifically importing `mount` rather than just calling `Enzyme.mount`...)